modules/carto/src/api/fetch-map.ts

```diff
@@ -298,23 +281,27 @@ export async function fetchMap({
   const baseUrl = buildPublicMapUrl({apiBaseUrl, cartoMapId});
   const errorContext: APIErrorContext = {requestType: 'Public map', mapId: cartoMapId};
   const map = await requestWithParameters({baseUrl, headers, errorContext, maxLengthURL});
+  const context: _FetchMapContext = {
```

Comment:
```suggestion
  const context: FetchMapContext = {
```

Why the underscore?

Reply:
Intended to imply 'internal' but not a strong preference – will change it!